src/pages/user/UserOrcid.tsx

```diff
@@ -77,13 +80,15 @@ const UserOrcid: FC = () => {
     if (!authority) {
       return;
     }
+    setIsRemovingOrcid(true);
     const updatedAuthority = await removeQualifierIdFromAuthority(
       authority.systemControlNumber,
       AuthorityQualifiers.ORCID,
       id
     );
     if (updatedAuthority.error) {
       dispatch(setNotification(updatedAuthority.error, NotificationVariant.Error));
+      setIsRemovingOrcid(false);
     } else if (updatedAuthority) {
       dispatch(setExternalOrcid(''));
```

Comment:
skal ikke `setIsRemovingOrcid(false);` være her også?

Reply:
ja, egentlig, men modalen lukkes etter den er ferdig med det som skjer i if'en her. Det er bare en state for å vise at knappen får en spinner. Når jeg tenker meg om så trengs den egentlig ikke å settes til false i den if'en der heller 😛